website/package.json

```diff
@@ -6,7 +6,10 @@
     "build": "docusaurus build",
     "prepublish-gh-pages": "npm run copy-appd && npm run copy-schemas && npm run copy-workbench",
     "publish-gh-pages": "docusaurus-publish",
-    "copy-schemas": "cpy ../src/context/schemas/*.schema.json static/schemas/next",
+    "copy-schemas-api": "cpy ../schemas/api static/schemas/next/api",
+    "copy-schemas-bridging": "cpy ../schemas/bridging static/schemas/next/bridging",
+    "copy-schemas-context": "cpy ../schemas/context static/schemas/next/context",
+    "copy-schemas": "npm run copy-schemas-api && npm run copy-schemas-bridging && npm run copy-schemas-context",
```

Comment:
I tried this before. Weirdly when you run `cpy ../schemas/api ../schemas/bridging ../schemas/context static/schemas/next/` it copies all files from the input folders to the destination folder rather than copying the input folders.

Reply:
I have just had to learn a shitload about cpy - the latest version has a number of breaking changes in it... try merging upstream/master into your branch and resolving conflicts then see below:

My current understanding is that this should work:
`"copy-schemas": "del-cli static/schemas/next/*/ && cpy \"../schemas/api\" \"../schemas/bridging\" \"../schemas/context\" static/schemas/next/",`